packages/rum/test/specs/index.spec.js

```diff
@@ -46,22 +46,6 @@ describe('index', function() {
     jasmine.DEFAULT_TIMEOUT_INTERVAL = originalTimeout
   })
 
-  it('should log only on browser environments', () => {
```

Comment:
What do you think about changing this test to require `index.js` twice and test for the right condition

Reply:
the test is moved to `bootstrap.spec.js`. Why do we need another test here?

Reply:
Just as a smoke test, and also, to test the actual scenario of including the agent twice which the bootstrap.spec.js doesn't test

Reply:
Ahh got it. yes will do 👍